nbs/04_learning.ipynb

```diff
@@ -58,6 +58,7 @@
     "#| exporti\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        train_loader = data_module.train_dataloader(next(keys), t_configs.batch_size)
</pre><p>You are passing a <code>PRNGKey</code> to the function, but it is supposed to be a seed number. Why? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>